package.json

```diff
@@ -96,6 +96,7 @@
 		"jest-puppeteer": "3.2.1",
 		"lerna": "3.0.0-rc.0",
 		"lint-staged": "7.2.0",
+		"lodash": "4.17.10",
```

Comment:
Why do we need both?

Reply:
Node 8 doesn't have ES modules enabled natively so we need to use `lodash` in all CLI tools.

Reply:
Looks like there's some breaking in e2e tests, I guess it's related?